Editor/UI/CreatorEditorStyles.cs

```diff
@@ -24,7 +23,7 @@ public static GUIStyle Title
                     title.fontSize = 24;
                     title.fontStyle = FontStyle.Bold;
                     title.normal.textColor = HighlightTextColor;
-                    title.padding = new RectOffset(Ident, BasePadding, Ident, Ident);
+                    title.margin = new RectOffset(Indent, BaseMargin, Indent, Indent);
```

Comment:
seems like on text margin does not have any effect:

![Unbenannt](https://user-images.githubusercontent.com/8425643/90379759-6cd94900-e07b-11ea-8d0b-d580bd5a4f54.png)

Reply:
Might be, but can we maybe merge everything and then do the styling for all pages as discussed before?